boards/arm/rpi_pico/rpi_pico-common.dtsi

```diff
@@ -32,6 +32,42 @@
 	aliases {
 		watchdog0 = &wdt0;
 	};
+
+	pico_header: connector {
+		compatible = "Raspberry-Pi-Pico-header";
```

Comment:
Please define and add  .yaml file into dts/bindings/gpio.
(Should not use capital character here. IMO, "raspberrrypi,pico-header" is preferable.)